### PR TITLE
configured Storybook 

### PR DESCRIPTION
**Pull request by umangag07**
Made two components heading and button to make stories.

### CODE REVIEW

File: src/App.sublime-workspace [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
```
Comment (digvijay-u) [inline src/App.sublime-workspace]: Pls add this file to the gitignore

File: src/Component/Button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React, { ReactNode } from "react";
+import "./button.css";
+export interface buttonProp {
```
Comment (digvijay-u) [inline src/Component/Button/Button.tsx]:
```suggestion
export interface ButtonProps {
```
The interfaces or types are usually starts with capital letter

File: src/Component/Button/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  onclick: () => void;
+}
+export const Button: React.FC<buttonProp> = ({
+  children,
+  type = "primary",
+  size = "medium",
+  onclick,
+}) => {
+  return (
+    <button className={`button ${type} ${size}`} onClick={onclick}>
```
Comment (digvijay-u) [inline src/Component/Button/Button.tsx]: We will use styled components eventually

**Review by digvijay-u — CHANGES_REQUESTED** [review]

File: src/App.sublime-workspace (hunk 1 above)
Comment (umangag07) [reply src/App.sublime-workspace]: okay I will do that.

File: src/Component/Button/Button.tsx (hunk 2 above)
Comment (umangag07) [reply src/Component/Button/Button.tsx]: okay

File: src/Component/Button/Button.tsx (hunk 3 above)
Comment (umangag07) [reply src/Component/Button/Button.tsx]: Yeah, that's good actually I was thinking to use it. I will change this with the styled component.

Comment (digvijay-u) [reply src/Component/Button/Button.tsx]: Don't change it just yet. I have created a ticket for it

**Review by digvijayu — APPROVED** [review]

**Pull request merged**